src/views/Timeline.vue

```diff
@@ -176,7 +176,7 @@ function selectState(e) {
           </template>
         </BaseDropdown>
       </div>
-      <div class="md:rounded-lg bg-skin-block-bg border">
+      <div class="md:rounded-lg bg-skin-block-bg md:border">
```

Comment:
Timeline now looks like this on darkmode, not sure why
<img width="691" alt="image" src="https://user-images.githubusercontent.com/51686767/159639876-cd5b898d-f185-4149-b00f-c9dfbd1df6bf.png">

Reply:
Thanks. Fixed that and left a comment in styles.scss.